.gitignore

```diff
@@ -1,3 +1,4 @@
+/.idea
```

Comment:
This should go into your global ignores list. Please remove it from here.

Reply:
removed